Owin.Security.Providers/LinkedIn/Provider/LinkedInAuthenticatedContext.cs

```diff
@@ -69,12 +70,17 @@ public LinkedInAuthenticatedContext(IOwinContext context, JObject user, string a
         /// </summary>
         public string Name { get; private set; }
 
-        public string Link { get; private set; }
+        /// <summary>
+        /// Get the user's first name
+        /// </summary>
+        public string GivenName { get; private set; }
 
         /// <summary>
-        /// Gets the LinkedIn username
+        /// Get the user's last name
         /// </summary>
-        public string UserName { get; private set; }
```

Comment:
Please do not remove things which can break other people's code. Put this property back